frontend/src/app/components/post-battle-page/team-info.tsx

```diff
@@ -7,11 +7,15 @@ export type TeamInfoProps = {
   players: PlayerWithStats[] | undefined;
 };
 export const TeamInfo = ({ title, players }: TeamInfoProps) => {
+
+  console.log("title", title);
+
+
   return (
     <div>
       <Group justify="space-between">
         <Text py={3} fw={'bold'} size={'xl'}>
-          {title}
+          {title || 'Team'}
         </Text>
         <Text py={3}>Score</Text>
       </Group>
```

Comment:
1. **Console Logging**: The `console.log("title", title);` statement is left in the production code, which can lead to performance issues and potential information leakage. Remove this line or replace it with a proper logging mechanism if needed.

2. **Undefined Players Handling**: The `players` prop is typed as `PlayerWithStats[] | undefined`, but there is no handling for the case when `players` is `undefined`. This could lead to runtime errors. Consider providing a default value or handling the undefined case explicitly:
   ```typescript
   const playerList = players || [];
   playerList.map((player) => { ... });
   ```

Quality Rate: 6/10